index.html

```diff
@@ -681,35 +697,54 @@ <h2>DID Documents</h2>
                 <code>WotThing</code> or <code>WotDirectory</code>, respectively, in the DID Document that 
                 the TD's identifier resolves to.</span>
             </p>
+            <p><span class="rfc2119-assertion" id="introduction-did-service-context">In order to define Service Endpoints
+                for WoT Discovery, the DID Document obtained by resolving the DID of a <a>Thing</a> or <a>Thing Description Directory</a>
+                MUST include the URL <code>https://www.w3.org/2021/wot/discovery/did</code> in its <code>@context</code> 
```

Comment:
This URL doesn't resolve, but I expect that will be fixed in time.

You might also want to specify that the URL must exist after the 1st array entry in the `@context` field. It should be obvious for those in the know, but it might not be... perhaps an example would help (I don't know if you already have one in the spec since I'm reviewing the PR instead of the human readable version).

Reply:
Thanks for your feedback.  I'm not really happy with the year-based-URLs either since we then ALSO have version numbers, and the year we allocated no longer aligns with the date of publication of our spec, etc. etc.  However, our main goal at present is to at least be consistent with our current set of publications.  Our other constraint is that we were previously using year-based URLs, some of which we will still be using, so...

Reply:
Anyway, we are working on fixing the URL and having it resolve to an appropriate ontology definition.